nixos/modules/virtualisation/digital-ocean-config.nix

```diff
@@ -0,0 +1,81 @@
+{ config, lib, pkgs, options, ... }:
+with lib;
+let
+  cfg = config.virtualisation.digitalOceanImage;
+in
+{
+  imports = [
+    ../profiles/qemu-guest.nix
+  ];
+
+
+  fileSystems."/" = {
+    device = "/dev/disk/by-label/nixos";
+    autoResize = true;
+    fsType = "ext4";
+  };
+
+  boot.growPartition = true;
+  boot.kernelParams = [ "console=ttyS0" "panic=1" "boot.panic_on_fail" ];
+  boot.initrd.kernelModules = [ "virtio_scsi" ];
+  boot.kernelModules = [ "virtio_pci" "virtio_net" ];
+
+  # password "nixos"
+  users.users.nixos = {
+    createHome = true;
+    hashedPassword = "$6$7g1GlQ/pgXdnVA$TEBczCVy4cuiHp8vpSeTLjottXDHDtwrUodhU9dGbeLtgE/fCeSvX.D27NstKTLJ2KBUignd5Kn8zdjeWRJTm0";
```

Comment:
It would be better to avoid that since a ssh key can be provided with cloud-init.